uui-components/src/pickers/PickerBase.tsx

```diff
@@ -169,10 +172,10 @@ export abstract class PickerBase<TItem, TId, TProps extends PickerBaseProps<TIte
     getView(): IDataSourceView<TItem, TId, any> {
         return this.props.dataSource.getView(this.getDataSourceState(), this.handleDataSourceValueChange, {
             getRowOptions: this.getRowOptions,
-            cascadeSelection: this.props.cascadeSelection,
             getSearchFields: this.props.getSearchFields || ((item: TItem) => [this.getName(item)]),
```

Comment:
@jakobz, @AlekseyManetov, have no idea, if this default is specific to the current component. I'm scared to align this option with `cascadeSelection` and `sortBy` to prevent overriding the options from the datasource. On the one hand, it is required if some value is provided at the datasource and not provided to the Picker, but on the other hand, if some users haven't specified getSearchFields and rely on the current behavior, it will break their current flow. WDYT? How can we perform this required change?

Reply:
Let's assume that for cases with PickerInput this props will be passed directly to the picker, because here we need to have a default behavior. Let's return to this moment when rework our Picker Input api.